track_oracle/file_formats/file_format_base.cxx

```diff
@@ -32,37 +32,45 @@ glob_to_regexp_string( const string& glob )
   // This code is lifted straight from vul_file_iterator.cxx.
   //
 
+  static const string meta_chars = "^$.[()|?+*\\";
+
   string baseglob = vul_file::basename(glob);
   string::iterator i = baseglob.begin();
   bool prev_slash=false, in_sqr_brackets=false;
   //assemble the Regexp string
   string re = "^"; // match the start of the string
   while (i != baseglob.end())
   {
-    if (*i=='\\' && !prev_slash)
+    const char& c = *i;
+    if (c =='\\' && !prev_slash)
       prev_slash = true;
```

Comment:
Should this have brackets around a single line block?

Reply:
Probably... I didn't update the style when I ported it over from vxl. Update coming shortly.